oem/ibm/libpldmresponder/oem_ibm_handler.cpp

```diff
@@ -1705,6 +1705,7 @@ void pldm::responder::oem_ibm_platform::Handler::processPowerOffSoftGraceful()
             << "Error in powering down the host. Unable to set property RequestedPowerTransition. ERROR="
             << e.what() << "\n";
     }
+    processPowerCycleOffSoftGraceful();
```

Comment:
I'm confused. I thought the issue was that PHYP was calling processPowerOffHardGraceful() to set the APR, and in that function, we need to add the call to hard power off (processPowerOffSoftGraceful())

Reply:
so the expectation from PHYP was they when they call processPowerOffHardGraceful(), we need to set the APR and also chassis off. And what we were already doing is setting the APR. 
Here we have just added code to do the Chassisoff  after the APR, and as processPowerCycleOffSoftGraceful() already does that we have just reused the code here.

Reply:
Ah I got what you were trying to say. @ArchanaKakani could you please move the processPowerCycleOffSoftGraceful() inside processPowerOffHardGraceful() after setting the APR please.